great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -235,20 +237,29 @@ class ExpectColumnValuesToBeBetween(ColumnMapExpectation):
         "strict_max",
     )
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
-        """
-        Validates that a configuration has been set, and sets a configuration if it has yet to be set. Ensures that
-        necessary configuration arguments have been provided for the validation of the expectation.
+        """Validates the configuration of an Expectation.
+
+        For `expect_column_values_to_be_between` it is required that:
+            - One of `min_value` or `max_value` is not `None`.
+            - `min_value` and `max_value` are one of the following types: `datetime`, `float`, `int`, or `dict`
+            - If `min_value` or `max_value` is a `dict`, it is assumed to be an Evaluation Parameter, and therefore the
+              dictionary keys must be `$PARAMETER`.
```

Comment:
A small nit, but I think the rendering will be more consistent with other files if it was un-tabbed to be in line with the `For`. 

![Screen Shot 2023-01-20 at 1 17 13 PM](https://user-images.githubusercontent.com/25670697/213806896-9c0cf7b1-9d67-46d0-8af8-852ae6bec2d0.png)

Looks like [this](https://docs.greatexpectations.io/docs/reference/api/great_expectations/util/)